src/models/luckyTickets/luckyTickets_test.go

```diff
@@ -0,0 +1,57 @@
+package luckyTickets
+
+import (
+	"testing"
+	"reflect"
+	"fmt"
+)
+
+func TestGetLuckyTickets(t *testing.T){
+	testObjects := []struct{
+		name string
+		needError bool
+		inputData context
```

Comment:
your `context` is misunderstood here.. see the comment above

Reply:
dont undestood.
https://i.imgur.com/HXfVpTK.png

Reply:
It is related to the previous comment. `context` is not a good name here. Someone could ask, why `inputData` is a `context`? In general they are separate things...